docs/extend/extend-apiml/api-mediation-caching-service.md

```diff
@@ -36,25 +37,30 @@ Information from cached APIs is stored as a JSON in the following format:
 ## Storage methods
 
 The Caching Service supports the following storage solutions, which provide the option to add custom implementation.  
+
+### Infinispan (recommended)
+
+Infinispan is a storage solution that can also run on the z/OS platform. It can store data structures in key-value pairs, has high-availability support, and is highly performant.
```

Comment:
"A storage solution that can also run on the z/OS platform. Infinispan can store ..."

Reply:
I made a mistake when I created this branch and, as a result, it now includes another writer's content. Since this is the work of another writer, I will not be editing this file.